CMakeLists.txt

```diff
@@ -468,9 +432,23 @@ if ((DISABLE_CURL STREQUAL "OFF" OR NOT DISABLE_CURL) AND NOT DISABLE_CIVET)
 	endif()
 endif()
 
+option(DISABLE_EXPRESSION_LANGUAGE "Disables the scripting extensions." OFF)
+if (NOT DISABLE_EXPRESSION_LANGUAGE)
+    createExtension(EXPRESSION-LANGUAGE-EXTENSIONS "EXPRESSION LANGUAGE EXTENSIONS" "This enables NiFi expression language" "extensions/expression-language" "extensions/expression-language/tests")
+    if(NOT USE_SYSTEM_OPENSSL OR USE_SYSTEM_OPENSSL STREQUAL "OFF")
+  add_dependencies(minifi-expression-language-extensions libressl-portable)
+endif()
+    if(NOT USE_SYSTEM_CURL)
+		message("minifi-expression-language-extensions will depend on curl-external")
+		add_dependencies(minifi-expression-language-extensions curl-external)
+	endif()
+endif()
+
+
+
 option(DISABLE_CIVET "Disables CivetWeb components." OFF)
 if (NOT DISABLE_CIVET)
-	createExtension(CIVETWEB CIVETWEB "This enables ListenHTTP" "extensions/civetweb" "${TEST_DIR}/civetweb-tests")
```

Comment:
test moved to standard processors. civetweb is an extension that really isn't an extension at all because it's required for every other extension. Could be part of core.